aten/src/ATen/native/mkldnn/Conv.cpp

```diff
@@ -919,6 +998,14 @@ TORCH_LIBRARY_IMPL(mkldnn, Meta, m) {
       TORCH_SELECTIVE_NAME("mkldnn::_convolution_transpose_pointwise"),
       TORCH_FN(mkldnn_convolution_transpose_pointwise_meta));
 }
+
+
+TORCH_LIBRARY_IMPL(mkldnn, Functionalize, m) {
+  m.impl(
+      TORCH_SELECTIVE_NAME("mkldnn::_convolution_pointwise_.binary"),
+      TORCH_FN(functionalization::mkldnn_convolution_pointwise_binary_));
+}
```

Comment:
@bdhirsh For inplace ops not registered via native_functions.yaml like this one, do we have an easier way of implementing the functionalization without explicit implementation op-by-op?

Reply:
Yes this is very boilerplate-y... Long term, we should write a generic / boxed functionalization kernel that works for arbitrary mutable/aliasing ops. But until then, writing a one-off kernel here seems acceptable.